sdk/quantum/quantum-jobs/src/quantumJobClientContext.ts

```diff
@@ -20,18 +20,23 @@ export class QuantumJobClientContext extends coreHttp.ServiceClient {
 
   /**
    * Initializes a new instance of the QuantumJobClientContext class.
+   * @param credentials Subscription credentials which uniquely identify client subscription.
    * @param subscriptionId The Azure subscription ID. This is a GUID-formatted string (e.g.
    *                       00000000-0000-0000-0000-000000000000)
    * @param resourceGroupName Name of an Azure resource group.
    * @param workspaceName Name of the workspace.
    * @param options The parameter options
    */
   constructor(
+    credentials: coreHttp.TokenCredential | coreHttp.ServiceClientCredentials,
     subscriptionId: string,
     resourceGroupName: string,
     workspaceName: string,
     options?: QuantumJobClientOptionalParams
   ) {
+    if (credentials === undefined) {
```

Comment:
```suggestion
    if (credentials == undefined) {
```
This would cover both `null` and `undefined`.

Reply:
this code is auto-generated by autorest so I'll open a bug there

Reply:
I see.